src/sentry/api/endpoints/project_event_details.py

```diff
@@ -72,7 +104,17 @@ def get(self, request, project, event_id):
         except IndexError:
             prev_event = None
 
+        try:
+            user_report = UserReport.objects.get(
+                event_id=event.event_id,
+                project=event.project,
+            )
+        except UserReport.DoesNotExist:
+            user_report = None
+
         data = serialize(event, request.user)
+        data['userReport'] = serialize(user_report, request.user)
+        data['release'] = self._get_release_info(request, event)
```

Comment:
This `userReport` and `release` stuff is just to make the payload returned by this endpoint the same as the `/events/` one

Reply:
would probably be nice to move the shared stuff between this and the other endpoint into a serializer or something (maybe `DetailedEventSerializer`?) but doesn't need to be part of this pr

Reply:
Good idea. Have done it here. https://github.com/getsentry/sentry/pull/10921